frontend/src/components/ParticipantsListComponent/PrevPointTableHead.js

```diff
@@ -0,0 +1,34 @@
+// TODO:
```

Comment:
This seems like it's a todo from another file?

Reply:
yup removed